### PR TITLE
[9.0] [MIG] stock_lot_scrap

### PR DESCRIPTION
**Pull request by pedrobaeza**
Relevant commit is 0f4e8bc235cd8d694b2db56792c2f1a326068543

cc @Tecnativa

### CODE REVIEW

File: stock_lot_scrap/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+==================
+
+This module adds a button in *Production Lot/Serial Number* view form to
+*Scrap* all quants contained.
+
+
+Configuration
+=============
+
+* Go to *Settings > Warehouse > Traceability*.
```
Comment (luismontalba) [inline stock_lot_scrap/README.rst]: * Go to *Inventory > Configuration > Settings*.

File: stock_lot_scrap/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* Enable *Track lots or serial numbers*.
+
+
+Usage
+=====
+
+* Go to *Warehouse > Traceability > Serial Numbers*, and enter in view form mode.
```
Comment (luismontalba) [inline stock_lot_scrap/README.rst]: * Go to *Inventory > Inventory Control > Serial Numbers / Lots*.

**Review by luismontalba — APPROVED** [review]
Functionality tested on runbot

**Review by chienandalu — APPROVED** [review]
tested on runbot

**Pull request merged**